src/Orleans.Runtime/GrainDirectory/LocalGrainDirectory.cs

```diff
@@ -501,14 +480,16 @@ public SiloAddress CalculateTargetSilo(GrainId grainId, bool excludeThisSiloIfSt
             int hash = unchecked((int)grainId.GetUniformHashCode());
 
             // excludeMySelf from being a TargetSilo if we're not running and the excludeThisSIloIfStopping flag is true. see the comment in the Stop method.
-            bool excludeMySelf = !Running && excludeThisSiloIfStopping;
+            // excludeThisSIloIfStopping flag was removed because we believe that flag complicates things unnecessarily. We can add it back if it turns out that flag 
+            // is doing something valuable. 
```

Comment:
Not sure we need a comment about dead code - the comment can be added to the PR comments instead

Reply:
I think it is useful to add something here in case this flag is useful somehow. Add in the pr would juts got lost after it merged